### ISSUE TITLE
HOOBS docker no longer working after update 

### ISSUE THREAD

**Issue by ps2sunvalley** [hoobs-org/HOOBS#554]
**Description**
After update to latest version (cannot tell exactly what version) docker does not work.

**Version**
Latest tag on docker
**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ X] Yes
- [ ] No

**Pervious version**
Pretty sure it was 3.1.23

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [ X] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [ X] Custom Install

**If this is a custom install, what's your operating system?**
Linux via docker on unRaid

**List your plugins**
-Nest
-Belkin WeMo

**Post your config**

* Remove any passwords from the config file. This is public.

```
{
    "server": {
        "port": 8282,
        "origin": "*",
        "autostart": 0,
        "home_setup_id": "X-HM://0023ISYWY08DL",
        "polling_seconds": 5,
        "docker": true
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hoobs-dark",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": "94040"
    },
    "bridge": {
        "name": "HOOBS",
        "port": 51826,
        "pin": "031-45-144",
        "username": "60:19:91:E6:45:4C"
    },
    "description": "",
    "ports": {},
    "plugins": [
        "nest",
        "homebridge-platform-wemo"
    ],
    "accessories": [],
    "platforms": [
        {
            "platform": "Nest",
            "plugin_map": {
                "plugin_name": "nest"
            },
            "googleAuth": {
                "issueToken": "token",
                "cookies": "cookiesO",
                "apiKey": "api key"
            }
        },
        {
            "platform": "BelkinWeMo",
            "name": "WeMo Platform",
            "plugin_map": {
                "plugin_name": "homebridge-platform-wemo"
            }
        }
    ],
    "package_manager": "npm",
    "system": "hoobs"
}
```

**Post your log**

```
Require stack:
- /hoobs/node_modules/glob/glob.js
- /hoobs/lib/cli.js
- /usr/src/hoobs/scripts/docker.js
- /usr/src/hoobs/bin/hoobs
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
at Function.Module._load (internal/modules/cjs/loader.js:864:27)
at Module.require (internal/modules/cjs/loader.js:1044:19)
at require (internal/modules/cjs/helpers.js:77:18)
at Object.<anonymous> (/hoobs/node_modules/glob/glob.js:47:16)
at Module._compile (internal/modules/cjs/loader.js:1158:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
at Module.load (internal/modules/cjs/loader.js:1002:32)
at Function.Module._load (internal/modules/cjs/loader.js:901:14)
at Module.require (internal/modules/cjs/loader.js:1044:19)
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)

(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

(node:1) UnhandledPromiseRejectionWarning: Error: Cannot find module 'inherits'
```

**Additional context**

**Comment by mkellsy** [comment]
Did you rebuild the container?

**Comment by mkellsy** [comment]
We are testing 3.1.27.

**Comment by mkellsy** [comment]
I just got done testing the new Docker image. We will release this tomorrow.
Thank you for your patience.

**Comment by ps2sunvalley** [comment]
> Did you rebuild the container?

I have not

**Comment by mkellsy** [comment]
With the rocker image you rebuild the container with the latest image from Docker Hub.

**Comment by mkellsy** [comment]
I just released 3.1.27
All the issues are now fixed.
I would recommend upgrading.

**Comment by ps2sunvalley** [comment]
> I just released 3.1.27
> All the issues are now fixed.
> I would recommend upgrading.

Upgraded this morning, all is fixed now!
